aiohttp/web_request.py

```diff
@@ -613,7 +613,7 @@ def body_exists(self) -> bool:
                 field_ct = field.headers.get(hdrs.CONTENT_TYPE)
 
                 if isinstance(field, BodyPartReader):
-                    if field.filename and field_ct:
+                    if field.filename:
```

Comment:
Could you please attach a comment pointing to the right place in the RFC to prevent this from happening in the future.

Reply:
Sure, I saw it. I just wanted a code comment here :)